dockers/actions-plan-preview/planpreview.go

```diff
@@ -161,6 +168,8 @@ func makeCommentBody(event *githubEvent, r *PlanPreviewResult) string {
 
 	changedApps, pipelineApps, quickSyncApps := groupApplicationResults(r.Applications)
 
+	var detailLen int64
```

Comment:
Not sure why do we need this var of type `int64`, is the `int` type not good enough? If we use `int` we can remove typecasting in the below implementation.

Reply:
I was wrong; I was confusing int with uint16.
thank you!